curations/nuget/nuget/-/StackExchange.Redis.yaml

```diff
@@ -42,24 +42,77 @@ revisions:
   1.2.6:
     licensed:
       declared: MIT
+  2.0.0-alpha.465:
+    licensed:
+      declared: MIT
   2.0.495:
     licensed:
       declared: MIT
+  2.0.510:
+    licensed:
+      declared: MIT
   2.0.513:
     licensed:
       declared: MIT
   2.0.519:
     licensed:
       declared: MIT
+  2.0.544:
+    licensed:
+      declared: MIT
+  2.0.558:
+    described:
+      releaseDate: '2019-04-03'
```

Comment:
This is not the publish date for this version. 2.0.558 is dated as 3/7/2019.

Reply:
@capfei - am I missing something, as I don't see some of these package versions actually published on Nuget.org.  I do not see a published package for 2.0.544 or 2.0558 or 2.0.510?

Reply:
They are unlisted:

- https://www.nuget.org/packages/StackExchange.Redis/2.0.510
- https://www.nuget.org/packages/StackExchange.Redis/2.0.544
- https://www.nuget.org/packages/StackExchange.Redis/2.0.558

Reply:
Unlisted versions do not show in the version list on NuGet.org so I have to add the version ID to the end of the address, like I did above.